tests/navigation.rs

```diff
@@ -147,3 +147,33 @@ fn can_navigate_using_breadcrumbs(
 
     Ok(())
 }
+
+#[rstest]
+#[case(server(&["--default-sorting-method", "date", "--default-sorting-order", "desc"]))]
+/// The breadcrumbs that we use to navigate contain the default sorting method and order
+fn can_specify_default_sorting_method_and_order(#[case] server: TestServer) -> Result<(), Error> {
+    // Create a vector of directory names. We don't need to fetch the file and so we'll
+    // remove that part.
+    let dir: String = {
+        let mut comps = DEEPLY_NESTED_FILE
+            .split('/')
+            .map(|d| format!("{d}/"))
+            .collect::<Vec<String>>();
+        comps.pop();
+        comps.join("")
+    };
+
+    let base_url = server.url();
+    let nested_url = base_url.join(&dir)?;
+
+    let resp = reqwest::blocking::get(nested_url.as_str())?;
+    let body = resp.error_for_status()?;
+    let parsed = Document::from_read(body)?;
+
+    // can go to intermediate dir with specified sort
+    let intermediate_dir_link =
+        get_link_from_text(&parsed, "very").expect("Intermediate dir link not found.");
+    assert_eq!("/very/?sort=date&order=desc", intermediate_dir_link);
+
```

Comment:
Wouldn't it be better to have a test that checks whether the default sorting works as expected? Sort it one way and then compare it against an expected order and then sort it another way and expect the order to be reversed.